### PR TITLE
KUP-33 Add endpoint to remove seeded data on testing

### PR DESCRIPTION
**Pull request by katyanna**
## Purpose
Seeded data should be easily deleted for testing purposes.

## Approach
Create DELETE endpoint for `seed`.

### CODE REVIEW

File: seeds/admin.py [hunk 1]
```diff
@@ -101,3 +100,18 @@ def provide_seed(request, organization_uuid: string, is_robot_test=False):
...
+            from seeds.data import robot_data as data
+        else:
+            from seeds.data import data
+
+        seed_env = SeedEnv(request, organization_uuid)
+        seed = SeedLogicModule(seed_env, data.SEED_DATA)
+        seed.clear_seed()
+
+        if is_robot_test:
+            return Response(status=status.HTTP_201_CREATED)
```
Comment (ralfzen) [inline seeds/admin.py]: please return here a `HTTP_204_NO_CONTENT`-status

File: seeds/data/robot_data.py [hunk 2]
```diff
@@ -645,48 +645,44 @@
...
+        # "start_date": f"{Monday}T08:40:00+02:00",
+        # "end_date": f"{Monday}T11:40:00+02:00",
+        # "type": "out-of-office",
+        # "address": "Abwesend",
+        # "siteprofile_uuid": None,
+        # "invitee_uuids": ["acca56ea-e088-4cd8-bcfa-80545bab2f95"],  # !!!!!!!!!
+        # "workflowlevel2_uuids": [],
+        # "contact_uuid": "a73ae6b9-66b2-4ae4-9f2c-d1765eb42869",
+        # "summary": "",
+    # },
```
Comment (ralfzen) [inline seeds/data/robot_data.py]: why is this commented out?

File: seeds/views.py [hunk 3]
```diff
@@ -10,20 +11,32 @@
...
+        )
+        # Todo: get messages from request
+        return response
+
+    elif request.method == "DELETE":
+        response = OrganizationAdmin.remove_seed(
+            request,
+            organization_uuid=str(organization.organization_uuid),
+            is_robot_test=True
+            # request, organization_uuid=str(organization.organization_uuid)
```
Comment (ralfzen) [inline seeds/views.py]: not needed

File: seeds/views.py [hunk 4]
```diff
@@ -10,20 +11,32 @@
...
+    # p.e. adapt OrganizationAdmin.provide_seed
 
-# ToDo: Write delete_seed method in OrganizationAdmin
+    if request.method == "POST":
+        response = OrganizationAdmin.provide_seed(
+            request,
+            organization_uuid=str(organization.organization_uuid),
+            is_robot_test=True,
+        )
+        # Todo: get messages from request
```
Comment (ralfzen) [inline seeds/views.py]: please pop also the messages from the request and put them into the response into a content_dict

File: seeds/views.py [hunk 5]
```diff
@@ -10,20 +11,32 @@
...
-    # p.e. adapt OrganizationAdmin.provide_seed
-
-    response = OrganizationAdmin.provide_seed(
-        request, organization_uuid=str(organization.organization_uuid), is_robot_test=True)
-    # Todo: get messages from request
+    organization, _ = Organization.objects.get_or_create(name="Robot Organization")
 
-    return response
+    request.META["HTTP_REFERER"] = "/"  # (hack for now) - think of better solution:
+    # p.e. adapt OrganizationAdmin.provide_seed
```
Comment (ralfzen) [inline seeds/views.py]: please take also care of this in the `provide_seed`-method. you could use `if request.META.get("HTTP_REFERER", False): redirect ...`

Comment (katyanna) [reply seeds/views.py]: Yes, I planned to take care of the pre-existing #todo comments in a different PR, but could be done here also (:

Comment (ralfzen) [reply seeds/views.py]: great, thanks!

File: seeds/data/robot_data.py (hunk 2 above)
Comment (katyanna) [reply seeds/data/robot_data.py]: the data is invalid for the seed, but I imagined it'd be better to leave it so @kutayy could check it.

**Pull request merged**